packages/@react-spectrum/cards/src/CardBase.tsx

```diff
@@ -94,7 +94,7 @@ function CardBase<T extends object>(props: CardBaseProps<T>, ref: DOMRef<HTMLDiv
     actionmenu: {UNSAFE_className: classNames(styles, 'spectrum-Card-actions'), align: 'end', isQuiet: true},
     footer: {UNSAFE_className: classNames(styles, 'spectrum-Card-footer'), isHidden: isQuiet},
     divider: {UNSAFE_className: classNames(styles, 'spectrum-Card-divider'), size: 'S'}
-  }), [titleProps, contentProps, height, isQuiet, orientation]);
+  }), [titleProps, contentProps, height, isQuiet, orientation, aspectRatioEnforce]);
```

Comment:
This one isn't quite right, aspectRatioEnforce isn't a memoized value, I'll go make a commit to address that

Reply:
Ah, thanks for catching that